CMakeLists.txt

```diff
@@ -1,20 +1,19 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 2.8.5)
+
+option(SAS "Whether to run static code analysis with SAS." OFF)
+if (SAS)
+    find_package(sas)
+    enable_sas(
+        FORMAT
+        COMPARISON_REPORT_DIR "${CMAKE_BINARY_DIR}/sas_report"
+        IGNORE_DIRS ${CMAKE_SOURCE_DIR}/build ${CMAKE_SOURCE_DIR}/cmake ${CMAKE_SOURCE_DIR}/doc
```

Comment:
What about moving ignoring the build dir (expressed in terms of CMAKE_BINARY_DIR) into `enable_sas`?